article/results.tex

```diff
@@ -56,59 +56,75 @@ \subsection{Best Practice Guidelines}
+It is worth to separate out at least such large steps as preprocessing, individual levels of analysis (e.g. per-subject vs whole-population), and article generation.
+YODA principles~\cite{yoda} again demonstrate one way how to organize digital artifacts across such stages for maximal re-usability.
+An good feature is to use a workflow platform which automatically checks for the presence of results from prior stages, and, if present, proceed to the next stage without triggering re-computation of prior stages.
+In this article we rely on simple GNU Make rules to provide very basic workflow across computational stages, but there exist hundreds of workflow engines and we refer reader to a recent call for FAIR workflows to discover more~\cite{fair-workflows}.
 
 \subsubsection{Container image size should be kept small}
 Due to a lack of persistency, addressing issues in container image contents requires rebuilding, which can be a time-consuming process.
 The smaller the container image, the easier it is.
 In particular, when using containers, it is thus advisable to \textit{not} provide data via a package manager or via manual download inside the build script.
-A suitable alternative is to assure provision of these resources outside of the container image, e.g. by bind-mounting Git (and DataLad) directories present on the host machine.
+A suitable alternative is to assure provision of these resources outside of the container image, e.g. by bind-mounting Git (and DataLad) directories present on the host machine or even combining them all via Git submodules mechanism as promoted by YODA principles~\cite{yoda}.
 
 \subsubsection{Containers should fit the scope of the underlying workflow steps}
-In order to not artificially extend the workload of rebuilding a container image, it is further advisable to not create a bundled container image if separate containers can be used for separate hierarchical steps of the workflow.
-Container image granularity is of course capped by article workflow granularity, but it is paramount to not compromise the former at the container level.
-At a minimum, as seen in this study, the article reexecution container image should be distinct from container images required for producing a summary meta-article.
+In order to not artificially extend the workload of rebuilding a container image, it is advisable to not create a bundled container image for sufficiently distinct separate steps of the workflow.
+For an example, as seen in this study, the article reexecution container image should be distinct from container images required for producing a summary meta-article.
+Complimentary, and initially possibly appearing contradictory, to the aforementioned advise we recommend to avoid building separate containers for related steps, in particular if supported by the same toolkits, and rather define and use different \emph{entry points} to the same container.
```

Comment:
I think you mean complementary, right?

Reply:
yes! hence 

```suggestion
Complementary, and initially possibly appearing contradictory, to the aforementioned advise we recommend to avoid building separate containers for related steps, in particular if supported by the same toolkits, and rather define and use different \emph{entry points} to the same container.
```

Reply:
wasn't supposed to be resolved -- we should accept/commit the suggestion